instaclustr/structs.go

```diff
@@ -149,24 +155,25 @@ type ClusterListItem struct {
-	CdcNetwork                    string          `json:"cdcNetwork,omitempty"`
-	Bundles                       []string        `json:"bundles,omitempty"`
-	ClientEncryption              bool            `json:"clientEncryption,omitempty"`
-	PasswordAuthentication        bool            `json:"passwordAuthentication,omitempty"`
-	UserAuthorization             bool            `json:"userAuthorization,omitempty"`
-	UsePrivateBroadcastRPCAddress bool            `json:"usePrivateBroadcastRPCAddress,omitempty"`
-	PrivateIPOnly                 bool            `json:"privateIPOnly,omitempty"`
-	Nodes                         []Node          `json:"nodes,omitempty"`
-	NodeCount                     int             `json:"nodeCount,omitempty"`
-	EncryptionKeyId               string          `json:"encryptionKeyId,omitempty"`
-	ResizeTargetNodeSize          string          `json:"resizeTargetNodeSize,omitempty"`
-	DataCentreRegion              string          `json:"dataCentre,omitempty" mapstructure:"data_centre_region"`
-	CdcStatus                     string          `json:"cdcStatus,omitempty"`
-	RackAllocation                *RackAllocation `json:"rackAllocation,omitempty" mapstructure:"rack_allocation,omitempty"`
+	ID                            string             `json:"id,omitempty"`
+	Name                          string             `json:"name" mapstructure:"name"`
+	CdcName                       string             `json:"cdcName,omitempty" mapstructure:"cdcName"`
+	Provider                      string             `json:"provider,omitempty"`
+	CdcNetwork                    string             `json:"cdcNetwork,omitempty"`
+	Bundles                       []string           `json:"bundles,omitempty"`
+	ClientEncryption              bool               `json:"clientEncryption,omitempty"`
+	PasswordAuthentication        bool               `json:"passwordAuthentication,omitempty"`
+	UserAuthorization             bool               `json:"userAuthorization,omitempty"`
+	UsePrivateBroadcastRPCAddress bool               `json:"usePrivateBroadcastRPCAddress,omitempty"`
+	PrivateIPOnly                 bool               `json:"privateIPOnly,omitempty"`
+	Nodes                         []Node             `json:"nodes,omitempty"`
+	NodeCount                     int                `json:"nodeCount,omitempty"`
+	EncryptionKeyId               string             `json:"encryptionKeyId,omitempty"`
+	ResizeTargetNodeSize          string             `json:"resizeTargetNodeSize,omitempty"`
+	DataCentreRegion              string             `json:"dataCentre,omitempty" mapstructure:"data_centre_region"`
+	CdcStatus                     string             `json:"cdcStatus,omitempty"`
+	RackAllocation                *RackAllocation    `json:"rackAllocation,omitempty" mapstructure:"rack_allocation,omitempty"`
+	PrivateLink                   *PrivateLinkConfig `json:"privateLink,omitempty" mapstructure:"private_link,omitempty"`
```

Comment:
**[QUESTION]** Why are we adding it here?

Reply:
Because we need PrivateLink in the datacentre's block from the response.

Reply:
Ah right-o, should we document somewhere? That eventhough we put PrivateLink in the CreateClusterRequest but it's actually a data centre config.

Reply:
Don't think so, we don't have any existing doc for this data centre config.